returnn/tf/util/data.py

```diff
@@ -5603,6 +5603,20 @@ def _default_feature_dim_axis(batch_dim_axis, time_dim_axis, batch_shape, sparse
   return axes[-1]
 
 
+def _get_most_general_dim_kind(dim_tags):
```

Comment:
I think the name is misleading. It's actually the most specific dim kind.

But I would better name it for what it is used for, ie like _get_merged_dim_kind.

Reply:
Yeah okay, that's better.